src/index.js

```diff
@@ -376,8 +382,9 @@ class MiniCssExtractPlugin {
                         '}',
                       ])
                     : '',
-                  'var head = document.getElementsByTagName("head")[0];',
-                  'head.appendChild(linkTag);',
+                  `var insert = ${insert};`,
+                  `if (typeof insert === 'function') { insert(linkTag); }`,
+                  `else { var target = document.querySelector(${insert}); target && insert === 'body' ? target && target.insertBefore(linkTag,target.firstChild) : target.appendChild(linkTag); } `,
```

Comment:
We should avoid extra logic here, if developer what to insert into body they should use `Function`

Reply:
@evilebottnawi so no string option at all? Only function

Reply:
Can amend the PR

Reply:
Yes, only functions

Reply:
updated, please re-review @evilebottnawi